fla/ops/path_attn/parallel.py

```
@@ -5,7 +5,7 @@
 
 import torch
 from einops import reduce
-
+import warnings
```

Ruff (0.11.9)

`warnings` imported but unused

Remove unused import: `warnings`

(F401)